meteor-app/imports/ui/pages/workspace/component.js

```diff
@@ -60,23 +63,32 @@ export default class WorkspacePage extends React.Component {
     }
   }
 
-  _rangeFilterOnChange (event) {
+  _rangeFilterOnChange (value) {
+    if (Date.now() - lastFilterChange < 50) {
```

Comment:
See https://lodash.com/docs/4.17.4#debounce for how to easily apply debouncing. Don't reinvent the wheel.